### ISSUE TITLE
✅ 🔗 🐞 Issue with Custom Gap Unit (Clamp) in Save Template and Cross Domain Copy-Paste (Duplicate of #24111)

### ISSUE THREAD

**Issue by warut2546** [elementor/elementor#29359]
### Prerequisites

- [ ] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

When using a Custom Gap Unit with the clamp() function in Elementor, the value is not properly retained in certain scenarios. Specifically:

**1. Saving Templates:** After saving a template that includes a custom gap unit defined with clamp(), re-importing or reusing the template results in the gap value being either null or completely missing.
**2. Cross Domain Copy-Paste:** When content with a custom gap unit (clamp()) is copied from one Elementor installation and pasted into another, the gap value fails to transfer correctly, leading to layout inconsistencies.

This issue causes the affected layouts to break, as the intended spacing is either removed or reset, making the design appear distorted.

### Steps to reproduce

1. Create a section or container in Elementor.
2. Set a Custom Gap Unit using the clamp() function (e.g., clamp(0.64rem, calc(0.35vw + 0.57rem), 0.88rem)).
3. Save the template or use the Cross Domain Copy-Paste feature to paste the content into another Elementor installation.
4. Open the saved template or review the pasted content.

### Expected behavior

The custom gap value, including the clamp() function, should be retained exactly as set when saving templates or performing cross-domain copy-paste operations. The layout should remain consistent.

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [ ] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.6.18
	PHP Version: 7.4.33
	PHP Memory Limit: 512M
	PHP Max Input Vars: 20000
	PHP Max Post Size: 1024M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.7.1
	Site URL: https://component.plookweb.com
	Home URL: https://component.plookweb.com
	WP Multisite: No
	Max Upload Size: 1 GB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: 7
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 3.1.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36

== Active Plugins ==
	All-in-One WP Migration
		Version: 7.87
		Author: ServMask

	Code Snippets
		Version: 3.6.5.1
		Author: Code Snippets Pro

	Dynamic.ooo - Dynamic Content for Elementor
		Version: 3.1.2
		Author: Dynamic.ooo

	Elementor
		Version: 3.25.9
		Author: Elementor.com

	Elementor Pro
		Version: 3.25.3
		Author: Elementor.com

	Exclusive Addons Elementor
		Version: 2.7.6
		Author: Exclusive Addons

	JetBlocks For Elementor
		Version: 1.3.15
		Author: Crocoblock

	JetElements For Elementor
		Version: 2.7.1.1
		Author: Crocoblock

	JetEngine
		Version: 3.5.7
		Author: Crocoblock

	JetEngine - Custom visibility conditions
		Version: 1.1.2
		Author: Crocoblock

	JetFormBuilder
		Version: 3.4.3
		Author: Crocoblock

	JetFormBuilder Stripe Gateway
		Version: 1.1.1
		Author: Crocoblock

	JetPopup
		Version: 2.0.7
		Author: Crocoblock

	JetSmartFilters
		Version: 3.5.7.1
		Author: Crocoblock

	Nextend Social Login
		Version: 3.1.15
		Author: Nextendweb

	Simple Custom CSS and JS
		Version: 3.49
		Author: SilkyPress.com

	User Role Editor
		Version: 4.64.2
		Author: Vladimir Garagulya

	WP Rocket
		Version: 3.17.0.2
		Author: WP Media

	WPS Hide Login
		Version: 1.9.17.1
		Author: WPServeur, NicolasKulka, wpformation


== Features ==
	Custom Fonts: 1
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Active by default
	Additional Custom Breakpoints: Active by default
	Container: Active by default
	Upgrade Swiper Library: Active by default
	Nested Elements Performance: Active by default
	Optimized Control Loading: Active
	Optimized Markup: Active
	Conditionally load Swiper CSS files: Inactive by default
	Plugin Onboarding: Active by default
	CSS Smooth Scroll: Active by default
	Default to New Theme Builder: Active by default
	Header &amp; Footer: Active by default
	Elementor Home Screen: Active by default
	Landing Pages: Active by default
	Nested Elements: Active
	Editor Top Bar: Active
	Pages Panel: Inactive by default
	Element Caching: Active
	Link In Bio: Active by default
	Floating Buttons: Active by default
	Elementor Editor Events: Inactive by default
	Atomic Widgets: Inactive by default
	Launchpad Checklist: Inactive by default
	Menu: Active


== Log ==
	
JS: showing 20 of 22JS: 2024-11-07 19:42:19 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view2156&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-07 19:42:29 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view4521&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-07 19:43:29 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view6880&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-07 19:43:31 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view6929&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-07 19:44:05 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view10539&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-07 19:55:19 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19958] View (cid: &quot;view30198&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-08 17:15:08 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view2621&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-08 17:15:08 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view2529&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-08 17:15:22 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view4132&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-08 17:15:22 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view4222&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-10 22:17:19 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view7996&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-10 22:17:19 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view8086&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-10 22:17:48 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view9892&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-10 22:57:40 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view2124&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-10 22:57:40 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view2214&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-14 16:45:32 [error X 11][../wp-content/plugins/elementor/assets/js/ai-media-library.min.js?ver=3.25.6:3:627442] (0 , x.createRoot) is not a function 
JS: 2024-11-14 19:44:33 [error X 33][../wp-content/plugins/elementor/assets/js/admin-top-bar.min.js?ver=3.25.6:2:1030] o.render is not a function 
JS: 2024-11-14 20:30:39 [error X 16][../wp-content/plugins/elementor/assets/js/app.min.js?ver=3.25.6:3:236099] l.render is not a function 
JS: 2024-11-14 21:35:08 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view6270&quot;) has already been destroyed and cannot be used. 
JS: 2024-11-15 05:47:52 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view3416&quot;) has already been destroyed and cannot be used. 

Log: showing 20 of 212024-11-14 22:18:12 [info] Elementor/Upgrades - _on_each_version Start  
2024-11-14 22:18:12 [info] Elementor/Upgrades - _on_each_version Finished 
2024-11-14 22:18:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.4',
  'to' => '3.25.6',
)]
2024-11-18 19:51:10 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.2',
  'to' => '3.25.3',
)]
2024-11-18 19:51:12 [info] elementor-pro::elementor_pro_updater Started 
2024-11-18 19:51:12 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-11-18 19:51:12 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-11-18 19:51:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.2',
  'to' => '3.25.3',
)]
2024-11-18 19:51:13 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.2',
  'to' => '3.25.3',
)]
2024-11-21 04:50:35 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.6',
  'to' => '3.25.8',
)]
2024-11-21 04:50:37 [info] elementor::elementor_updater Started 
2024-11-21 04:50:37 [info] Elementor/Upgrades - _on_each_version Start  
2024-11-21 04:50:37 [info] Elementor/Upgrades - _on_each_version Finished 
2024-11-21 04:50:37 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.6',
  'to' => '3.25.8',
)]
2024-11-21 04:50:38 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.6',
  'to' => '3.25.8',
)]
2024-11-21 05:44:03 [info] elementor::elementor_updater Started 
2024-11-21 05:44:03 [info] Elementor/Upgrades - _on_each_version Start  
2024-11-21 05:44:03 [info] Elementor/Upgrades - _on_each_version Finished 
2024-11-21 05:44:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.8',
  'to' => '3.25.9',
)]
2024-11-21 05:44:04 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.8',
  'to' => '3.25.9',
)]



== Elementor - Compatibility Tag ==
	
	Dynamic.ooo - Dynamic Content for Elementor: Incompatible
	Elementor Pro: Compatible
	Exclusive Addons Elementor: Compatible
	JetBlocks For Elementor: Incompatible
	JetElements For Elementor: Incompatible

== Elementor Pro - Compatibility Tag ==
	
	Dynamic.ooo - Dynamic Content for Elementor: Incompatible
	Exclusive Addons Elementor: Compatible
	JetBlocks For Elementor: Incompatible
	JetElements For Elementor: Incompatible

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello @warut2546!

Thank you for your submission! 🙏 

### 🔗 🐞 This issue is a Duplicate of #24111 

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved with the release of Elementor v3.26.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
